spec/spec_helper.rb

```diff
@@ -14,7 +14,7 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-Dir["./spec/support/**/*.rb"].sort.each {|f| require f}
+Dir['./spec/support/**/*.rb'].sort.each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.